chapters/synchronous.tex

```diff
@@ -1083,6 +1083,14 @@ \subsection{Clocked Discrete-Time and Clocked Discretized Continuous-Time Partit
 Such a partition has to be solved with a \emph{solver method} of \cref{solver-methods}.
 When \lstinline!previous(x)! is used on a continuous-time state variable \lstinline!x!, then \lstinline!previous(x)! uses the start value of \lstinline!x! as value for the first clock tick.
 
+The use of the operator \lstinline!sample! from \cref{event-related-operators-with-function-syntax} in a clocked partition is problematic.
+Diagnostics are appropriate especially if it is intended to generate sampling faster than the clocks, and otherwise the sampling should ideally be adjusted to the clock ticks.
```

Comment:
We should avoid introducing a new strength level of diagnostics.  Can we reformulate using _recommended_?  (Please check suggestion carefully, as it also suggests some other changes.)
```suggestion
A diagnostic is recommended, especially if the operator is intended to generate events faster than the clock ticks, and otherwise the sampling should ideally be adjusted to the clock ticks.
```